src/pages/en/guides/styling.md

```diff
@@ -4,14 +4,23 @@ title: Styling & CSS
 description: Learn how to style components with Astro.
 ---
 
-Astro includes special handling to make writing CSS as easy as possible. Styling inside of Astro components is done by adding a `<style>` tag anywhere.
 
-## Astro component styles
+INTRO BLAH BLAH: Astro includes special handling to make writing CSS as easy as possible. You can use your favorite libraries, write your own css files, or take advantage of Astro's own scoped styling to target UI frameworks at the component level.
```

Comment:
Would be cool if I’m wrong, but I’m pretty sure style scoping *only* works for Astro components. There may be ways to get something similar for other frameworks, but I usually think of Astro’s scoping feature as scoping at the `.astro` file level.